admin-portal/src/pages/aidPackage/assignSuppliers/assignSuppliers.css

```diff
@@ -20,6 +20,10 @@
   top: 3.5rem;
 }
 
+.need-column {
+  width: 15rem !important;
```

Comment:
Jaye didn't like using !important in a previous PR

Reply:
yeah, but it was for a grid. I tried few times didn't work without important